src/main/java/bibliotecame/back/Extension/ExtensionController.java

```diff
@@ -1,5 +1,6 @@
 package bibliotecame.back.Extension;
 
+import bibliotecame.back.ErrorMessage;
```

Comment:
Again I leave my comment here cause the related lines were not modified.

Note that there are some endpoints that are not handling runtime errors, for example, `approveExtension` looks for the extension with the `findById` method of the service, but this method throws an exception if the extension doesn't exist. This exception is not handled by the controller, and it should be.

Look for all the places in the app (not only this file) where this is happening and fix it.